plugins/module_utils/network/eos/argspec/logging_global/logging_global.py

```diff
@@ -228,7 +228,7 @@ def __init__(self, **kwargs):
                                     "type": "str",
                                     "choices": ["tcp", "udp"],
                                 },
-                                "port": {"type": "int"},
+                                "port": {"type": "int", "default": 514},
```

Comment:
Question:  Do I need to also submit a PR to update https://github.com/ansible-network/resource_module_models/blob/master/models/eos/logging_global/eos_logging_global.yml ?

Reply:
yes @bewing